includes/admin/load.php

```diff
@@ -340,13 +340,23 @@ function perflab_print_features_page_style() {
  */
 function perflab_plugin_admin_notices() {
 	if ( ! current_user_can( 'install_plugins' ) ) {
-		wp_admin_notice(
-			esc_html__( 'Due to your site\'s configuration, you may not be able to activate the performance features, unless the underlying plugin is already installed. Please install the relevant plugins manually.', 'performance-lab' ),
-			array(
-				'type' => 'warning',
-			)
-		);
-		return;
+		$are_all_plugins_active = true;
+		foreach ( perflab_get_standalone_plugin_version_constants() as $plugin_slug => $constant_name ) {
+			if ( ! defined( $constant_name ) ) {
```

Comment:
Wait, this checks whether all plugins are _active_, but what we need to check is whether they are all already _installed_.

Reply:
Right, this needs to rather check if `$plugin_slug` is among the installed plugins:

```suggestion
			if ( ! in_array( $plugin_slug, $installed_plugin_slugs, true ) ) {
```